base/Dockerfile

```diff
@@ -5,6 +5,7 @@ RUN apt-get update \
     ca-certificates=20190110 \
     lib32gcc1=1:8.3.0-6 \
     lib32stdc++6=8.3.0-6 \
+    lib32z1 \
```

Comment:
Please define the exact version like the other dependencies and install it only on the SourceMod image.